### PR TITLE
Review my projects

### PR DESCRIPTION
**Pull request by avill152**

### CODE REVIEW

File: Code/api/routes/projectsRoutes.js [hunk 1]
```diff
@@ -28,8 +28,9 @@ module.exports = function(app, express) {
     apiRouter.route('/projects/:id')
         .put(function (req, res) {
             Project.findById(req.params.id, function(err, proj){
+                console.log(req.body)
```
Comment (ghost) [inline Code/api/routes/projectsRoutes.js]: remove the console.log

File: Code/webapp/features/project-proposals/ProjectProposalController.js [hunk 2]
```diff
@@ -40,22 +39,28 @@ angular.module('ProjectProposalController', ['ProjectProposalService'])
...
-        function submit () {
+        
+        $scope.save = function save() {
+            console.log($scope.project.description)
+            console.log('testing save function')
             if(!vm.editingMode){
                 ProjectService.createProject($scope.project)
-                    .success(function(data){
+                    .then(function(data){
+                        console.log("Got here");
```
Comment (ghost) [inline Code/webapp/features/project-proposals/ProjectProposalController.js]: remove console.log

File: Code/webapp/features/project-proposals/ProjectProposalController.js [hunk 3]
```diff
@@ -40,22 +39,28 @@ angular.module('ProjectProposalController', ['ProjectProposalService'])
...
                     });
             }
             else{
                 ProjectService.editProject($scope.project)
-                    .success(function(data){
+                    .then(function(data){
+                        console.log("Got here");
```
Comment (ghost) [inline Code/webapp/features/project-proposals/ProjectProposalController.js]: remove console.log

File: Code/webapp/features/project-proposals/ProjectProposalController.js [hunk 4]
```diff
@@ -40,22 +39,28 @@ angular.module('ProjectProposalController', ['ProjectProposalService'])
...
                     });
             }
-        }
+        };
 
         $scope.toggleCheckbox = function toggleSelection(majors) {
+            console.log('testing checkbox')
```
Comment (ghost) [inline Code/webapp/features/project-proposals/ProjectProposalController.js]: remove console.log

File: Code/webapp/features/vip-projects/vip-projects-detailed.html [hunk 5]
```diff
@@ -1,6 +1,6 @@
 <div class="projects-top-bar">
     <div>
```
Comment (ghost) [inline Code/webapp/features/vip-projects/vip-projects-detailed.html:2]: if this div isn't needed. remove it

File: Code/webapp/features/project-proposals/projectProposal.html [hunk 6]
```diff
@@ -11,7 +11,8 @@
         </div>
         <div class="form-group">
             <label>*Team Description:</label>
-            <textarea class="form-control" rows="5" ng-model="project.description" required="True"></textarea>
+            <div text-angular ng-model="project.description"></div>
+            <!-- <textarea class="form-control" rows="5" ng-model="project.description" required="True"></textarea> -->
```
Comment (ghost) [inline Code/webapp/features/project-proposals/projectProposal.html]: if this commented line isn't needed, remove it

File: Code/webapp/features/vip-projects/vip-projects-detailed.html (hunk 5 above)
Comment (avill152) [inline Code/webapp/features/vip-projects/vip-projects-detailed.html:2]: That was done by rodolfo, i dont know if he needs it or not

**Comment by ghost** [comment]
looks good

**Pull request merged**